.golangci.yml

```
@@ -1,3 +1,6 @@
+issues:
+  # don't skip warning about doc comments
+  exclude-use-default: false
```

The `exclude-use-default` setting [ignores a lot more than just go doc comments](https://golangci-lint.run/usage/configuration/). So disabling it means we'll start seeing errors for the godoc comments (good!)... and other lint errors (bad!).

controller-runtime uses the follow setting, which seems like a reasonable middle ground.
 
```suggestion
issues:
  # don't exclude the default set of lint 
  exclude-use-default: false

  # restore some of the defaults
  # (fill in the rest as needed)
  exclude-rules:
  - linters: [errcheck]
    text: "Error return value of .((os\\.)?std(out|err)\\..*|.*Close|.*Flush|os\\.Remove(All)?|.*printf?|os\\.(Un)?Setenv). is not checked"

```